web/src/pages/home/homePage.js

```diff
@@ -651,7 +620,7 @@ const Homepage = () => {
         <div className="seasons-container">
           {seasons.map((season, index) => (
             <div className="season-card" key={index}>
-              <a href="https://www.domits.com/home/" target="_blank" rel="noopener noreferrer">
+              <a href="https://www.domits.com/home/" rel="noopener noreferrer">
```

Comment:
Dit moet global zijn, niet specifiek naar domits.com
```suggestion
              <a href="https://www.domits.com/home/" rel="noopener noreferrer">
```